### PR TITLE
fix: using index avoid uniqueid

### PR DESCRIPTION
**Pull request by emersonlaurentino**
related with https://github.com/vtex/order-items/pull/31

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## Fixed
```
Comment (emersonlaurentino) [inline CHANGELOG.md]:
```suggestion
### Fixed
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- using index avoid uniqueid
```
Comment (lariciamota) [inline CHANGELOG.md]:
```suggestion
- Quantity update by using item index instead of uniqueid
```

File: react/ProductList.tsx [hunk 3]
```diff
@@ -56,7 +56,7 @@ const ProductList: FC<Props> = ({ renderAsChildren, classes }) => {
         })
       }
 
-      updateQuantity({ uniqueId, quantity }, options)
+      updateQuantity({ index: item.index, quantity }, options)
```
Comment (lariciamota) [inline react/ProductList.tsx:59]: Is it safe to say that `index` always exists?

Comment (emersonlaurentino) [reply react/ProductList.tsx:59]: yep, because the uniqueId is transformed in an index into the API

**Review by lucasecdb — APPROVED** [review]

**Review by lariciamota — APPROVED** [review]

**Pull request merged**